content/en/docs/tasks/debug/debug-application/determine-reason-pod-failure.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Determine the Reason for Pod Failure
 content_type: task
+weight: 30
```

Comment:
It would have the same weight as this page for information https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/debug/debug-application/debug-statefulset.md.

I guess it could be better to not rely on alphabetical order for weight (because of localized version).

Reply:
I agree with you. I also think that the weight of [the article](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/debug/debug-application/debug-statefulset.md.) is not equivalent to the weight of this article. 
It is recommended to adjust it from [the article](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/debug/debug-application/debug-statefulset.md.) with fewer words.